### PR TITLE
change menu and overall design

### PR DESCRIPTION
**Pull request by janno42**
Lots of fun changes in the layout. Now only one logo and a yellowish background without any gradient. The content area is plain white. It features a much slighter menu with only one line (controlled overflow for many entries) and dropdown submenus. Round corners have been removed completely. The final product looks like this:
page

Menu items open on hover. Active pages are highlighted in the menu.
menu

Login/Logout and content functions have been moved to a separate menu. It's white (light gray) when not logged in, yellow when logged in.
usermenu_off
usermenu

The footer also got a redesign and features a smaller version of the infamous gradient bar (in reverse).
footer

The menu collapses on smaller screens and is then displayed as a list.
mobilemenu

### CODE REVIEW

File: _1327/templates/base.html [hunk 1]
```diff
@@ -84,15 +117,17 @@
...
 	{% endblock %}
 
 	<footer>
+		<div class="footer-line"></div> {# Top line for footer #}
+		<div class="footer-line gradient gradient-inv"></div> {# Gradient line #}
 		<div class="container">
 			<span class="pull-left">
 				{% for item in footer %}
-					<a href="{{ item.get_url }}">{{ item.title }}</a>
+					<li><a href="{{ item.get_url }}">{{ item.title }}</a></li>
```
Comment (Nef10) [inline _1327/templates/base.html]: maybe put the list items inside a `<ul>`?

Comment (janno42) [inline _1327/templates/base.html]: Okay

File: _1327/templates/base.html [hunk 2]
```diff
@@ -6,64 +6,97 @@
...
+				<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse" aria-expanded="false">
 					<span class="sr-only">Toggle navigation</span>
 					<span class="icon-bar"></span>
 					<span class="icon-bar"></span>
 					<span class="icon-bar"></span>
 				</button>
 			</div>
-			<div class="collapse navbar-collapse" id="top-navbar-collapse">
+
+			{# Main menu (for wider screens) #}
```
Comment (invliD) [inline _1327/templates/base.html]: Please combine these two versions of the menu into one.

**Comment by invliD** [comment]
:-1:

File: _1327/templates/base.html (hunk 2 above)
Comment (janno42) [inline _1327/templates/base.html]: see #262

File: _1327/templates/base.html [hunk 3]
```diff
@@ -6,64 +6,97 @@
...
+								<li><a href="{% url 'user_management.views.login' %}">Login</a></li>
+							{% endif %}
+							{% if CAN_CREATE_INFORMATIONPAGE %}
+								<li role="separator" class="divider"></li>
+								<li class="disabled"><a href="#">{% trans "Create content" %}</a></li>
+								<li><a href="{% url 'information_pages:create' %}">{% trans "Information page" %}</a></li>
+								<li><a href="{% url 'minutes:create' %}">{% trans "Minutes" %}</a></li>
+							{% endif %}
+						</ul>
+					</li>
```
Comment (Nef10) [inline _1327/templates/base.html]: add admin link

File: _1327/templates/base.html [hunk 4]
```diff
@@ -6,64 +6,97 @@
...
+								<li class="disabled"><a href="#">{% trans "Logged in as" %} {{ user.username }}</a></li>
+								<li><a href="{% url 'user_management.views.logout' %}">Logout</a></li>
+							{% else %}
+								<li><a href="{% url 'user_management.views.login' %}">Login</a></li>
+							{% endif %}
+							{% if CAN_CREATE_INFORMATIONPAGE %}
+								<li role="separator" class="divider"></li>
+								<li class="disabled"><a href="#">{% trans "Create content" %}</a></li>
+								<li><a href="{% url 'information_pages:create' %}">{% trans "Information page" %}</a></li>
+								<li><a href="{% url 'minutes:create' %}">{% trans "Minutes" %}</a></li>
```
Comment (Nef10) [inline _1327/templates/base.html]: check permission for each type

File: _1327/templates/base.html [hunk 5]
```diff
@@ -6,64 +6,108 @@
...
+			{# Toggle menu (for smaller screens) #}
+			<div class="navbar-collapse-menu" id="navbar-collapse" aria-expanded="false">
+				<ul class="">
+					{% if user.is_authenticated %}
+						<li class="disabled"><a href="#" disabled>{% trans "Logged in as" %} {{ user.username }}</a></li>
+						<ul><li><a href="{% url 'logout' %}">Logout</a></li></ul>
+					{% else %}
+						<li><a href="{% url 'login' %}">Login</a></li>
+					{% endif %}
+					{% if CAN_CREATE_INFORMATIONPAGE %}
```
Comment (Nef10) [inline _1327/templates/base.html]: please adjust permissions and add admin menu also in the collapsed menu

**Comment by Bartzi** [comment]
sad that the gradient isn't there anymore...
but besides that: :+1:

**Pull request merged**